oxidation/libparsec/crates/protocol/schema/authenticated_cmds/pki_enrollment_accept.json

```diff
@@ -0,0 +1,52 @@
+[
+    {
+        "label": "PkiEnrollmentAccept",
```

Comment:
The PKI commands has been introduced in API `3.2`/`2.8`, do we have a `introduced_in` field at command level to express this ? (just like we have `introduced_in` in `other_fields` to express the introduction of a new field in a existing command)

Reply:
No we don't.
I could add that field on the protocol level, 
but that would be used only for documentation purpose since the list of available commands are generated using the major versions (or that would required a specific module for a specific `major.minor` that would be alike to the `major` module without those commands.).

Reply:
> I could add that field on the protocol level, but that would be used only for documentation purpose

yes I think it would be good to have this, having a good documentation on what API version contains what is important when tracking bugs or considering deprecating an API version

Reply:
done